modules/apps/roles/roles-test/src/testIntegration/java/com/liferay/roles/service/test/RoleLocalServiceTest.java

```diff
@@ -99,6 +101,19 @@ public static void tearDownClass() {
 			_resourcePermission);
 	}
 
+	@Test
+	public void testAddRoleWithLongDescription() throws Exception {
+		Role role = RoleLocalServiceUtil.addRole(
+			TestPropsValues.getUserId(), null, 0, RandomTestUtil.randomString(),
+			null,
+			HashMapBuilder.put(
+				LocaleUtil.US, RandomTestUtil.randomString(4000) + "_US"
+			).build(),
+			RoleConstants.TYPE_REGULAR, null, null);
+
+		Assert.assertNotNull(role);
+	}
+
```

Comment:
@Kvale1733, please see the below suggestions:
1. Use `_roleLocalService`, instead of `RoleLocalServiceUtil`
2. Use `_role`, instead of `Role role`. Because the _role filed has `@DeleteAfterTestRun`. It can make sure the data will be deleted from the database after the test finished.
3. Please rename the test method name for testAddRoleWithDescriptionTextType